package.json

```diff
@@ -25,6 +30,7 @@
     "webpack": "^2.2.1"
   },
   "devDependencies": {
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
```

Comment:
Can you explain why this and the other related changes are relevant to this PR?

Reply:
This is needless to fastboot support. @marucjmar you can removes this dependency?